profiles/models.py

```diff
@@ -36,12 +36,12 @@ class Profile(models.Model):
 
 
 class Activity(models.Model):
-    activity_id = models.IntegerField(primary_key=True)
+    activity_id = models.AutoField(primary_key=True)
```

Comment:
In Django you shouldn't declare PK in case you need and autoincremental PK. Probably it's not possible to get rid of this in case if it's already there, just saying for the future.